.projenrc.js

```diff
@@ -8,6 +8,7 @@ const project = new TypeScriptProject({
     'aws-sdk',
     'minimist',
     'proxy-agent',
+    'camelcase',
   ],
   bin: {
     'cfn-import': 'lib/index.js',
```

Comment:
I would recommend separating the library use from the CLI use.

Usually the library entry point is `lib/index`, so I would put the cli under `lib/cli.js` or something like that.